src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -108,10 +116,9 @@ const formConfig = {
   }/v0/disability_compensation_form/submit_all_claim`,
   submit: submitForm,
   trackingPrefix: 'disability-526EZ-',
-  // formId: '21-526EZ-all-claims',
-  formId: '21-526EZ', // To test prefill, we'll use the 526 increase form ID for now
-  version: 1,
-  migrations: [],
+  formId: '21-526EZ',
+  version: migrations.length,
```

Comment:
The form version will always be the length of the migrations until we remove some.

The reason `version` is separate from `migrations` is because we were thinking we'd remove them later. The current implementation of SiP migrations assumes they'll never get removed, though, so this is likely to stay like this indefinitely. If we do later remove old migrations to reduce the bundle size, we'll likely replace them with `savedForm => savedForm`. :man_shrugging:

Reply:
Huh. TIL.